src/Symfony/Component/HttpKernel/Bundle/BundleInterface.php

```diff
@@ -56,6 +56,8 @@ public function getContainerExtension();
      * bundle.
      *
      * @return string The Bundle name it overrides or null if no parent
+     *
+     * @deprecated This method is deprecated as of 3.4 and will be removed in 4.0.
```

Comment:
Do we need to drop the final dot here according to the latest CS changes?

Reply:
Most @deprecated tags have a dot at the end currently. As this is a sentence, I think that's the way it should be.